lib/common/exceptions/method_not_allowed.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Common
```

Comment:
currently all of these common exceptions have a unique spec file. Would you mind adding one for this PR. Can copy and paste most of it -- I will likely refactor in the future into "shared examples"